merge_helm_values/__init__.py

```diff
@@ -62,16 +62,31 @@ def main():
         help=f"Skip the pre-commit hooks, default is {default_pre_commit_skip}",
         default=default_pre_commit_skip,
     )
+    parser.add_argument(
+        "--ignore-folder",
+        help="Ignore the folder when looking for the Helmfile files",
```

Comment:
how shall the input be formatted in a shell so that this argument is correctly parsed? something like `--ignore-folder '[folder_a, folder_b]'` with `' '` escaping? Can you provide an example in the readme?